oauth2client/contrib/gce.py

```diff
@@ -16,60 +16,117 @@
 
 Utilities for making it easier to use OAuth 2.0 on Google Compute Engine.
 """
-
```

Comment:
Maintain one-blank-line-between-module-doc-string-and-import-statements, please.